pkg/cloud/rgraph/rnode/action_delete.go

```diff
@@ -79,7 +79,7 @@ func (a *genericDeleteAction[GA, Alpha, Beta]) Run(
 		events = append(events, exec.NewDropRefEvent(ref.From, ref.To))
 	}
 
-	a.end = a.start
+	a.end = time.Now()
```

Comment:
I'd suggest moving this to the top of the function as a `defer` statement

Reply:
Do *not* use defer for setting variables. It leads to extremely hard to find bugs.